call_handlers/debugtools_commands.lua

```diff
@@ -618,4 +618,22 @@ function Commands:print_ai_stack_command(session, response, entity)
    response:resolve({'reported traceback for entity '..tostring(entity)})
 end
 
+function Commands:add_conversation_subject_command(session, response, entity, subject, polarity)
+   local conversation_component = entity:add_component('stonehearth:conversation')
+   local subject = conversation_component:add_conversation_subject(subject, polarity)
+   response:resolve(subject)
+end
+
+function Commands:get_conversation_subject_command(session, response, entity, subject)
+   local conversation_component = entity:add_component('stonehearth:conversation')
```

Comment:
Good point. I considered that but kind of wanted to just add it to anything to test. But now that we're formalizing how it should work, changing it to work the way you describe is better. Thanks!